jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt

```diff
@@ -26,9 +27,10 @@ import software.aws.toolkits.jetbrains.utils.TaggingResourceType
 import software.aws.toolkits.jetbrains.utils.toHumanReadable
 import software.aws.toolkits.resources.message
 
-class CloudFormationServiceNode(project: Project) : AwsExplorerServiceRootNode(project, message("explorer.node.cloudformation")) {
-    override fun serviceName() = CloudFormationClient.SERVICE_NAME
-
+class CloudFormationServiceNode(
+    project: Project,
+    explorerService: AwsExplorerService
```

Comment:
What's the value of passing this in vs having it be part of the `CloudFormationServiceNode` implementation? Is there a time where `explorerService` would be anything other than `AwsExplorerService. CLOUDFORMATION` ?

Reply:
That....is a great question. Got blinders on and didnt see the obvious thing.